package.json

```diff
@@ -28,7 +29,8 @@
     "framer-motion": "^10.8.4",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-to-html-element": "^1.4.7"
+    "react-to-html-element": "^1.4.7",
+    "react-toastify": "^9.1.2"
```

Comment:
I have the same feeling about if we need toast as of now. Another thing that could be discussed with the design team @pabloi-nf.